modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -0,0 +1,249 @@
+
+When you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively. When you run `rosa create cluster --interactive` at cluster creation time, you are presented with a series of interactive prompts that enable you to customize your deployment. For more information, see _Interactive cluster creation mode reference_.
+
+There are two `rosa` CLI modes for deploying a cluster with STS:
+
+* `manual` mode. With this mode, the `rosa` CLI generates the `aws` commands needed to create the required AWS Identity and Access Management (IAM) roles and policies, and an OpenID Connect (OIDC) provider. The corresponding policy JSON files are also saved to the current directory. This enables you to review the details before running the `aws` commands manually.
+* `auto` mode. You can use this mode to immediately create the required AWS Identity and Access Management (IAM) resources using the current AWS account.
+
+[IMPORTANT]
+====
+Only public and AWS PrivateLink clusters are supported with STS. Regular private clusters (non-PrivateLink) are not available for use with STS.
+====
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installations.
+====
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host.
+* If you are using a customer-managed AWS Key Management Service (KMS) key for encryption, you have created a symmetric KMS key and you have the key ID and Amazon Resource Name (ARN). For more information about creating AWS KMS keys, see link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html[the AWS documentation].
++
+[NOTE]
+====
+To successfully install ROSA 4.10 clusters, use ROSA CLI 1.1.11 or above.
+====
+
+.Procedure
+
+. Create the required account-wide roles and policies, including the Operator policies:
+.. Generate the IAM policy JSON files in the current working directory and output the `aws` CLI commands for review:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode manual <1>
+----
+<1> `manual` mode generates the `aws` CLI commands and JSON files needed to create the account-wide roles and policies. After review, you must run the commands manually to create the resources.
++
+.. After review, run the `aws` commands manually to create the roles and policies. Alternatively, you can run the preceding command using `--mode auto` to run the `aws` commands immediately.
+
+. (Optional) If you are using your own AWS KMS key to encrypt the control plane data volumes and the persistent volumes (PVs) for your applications, add the ARN for the account-wide installer role to your KMS key policy.
```

Comment:
This is existing content, but I noticed it while reading through:
s/(Optional)/Optional:

Reply:
@bmcelvee thanks for highlighting this. Resolved.